nyx_client/nyx_client/client.py

```diff
@@ -739,4 +764,6 @@ def unsubscribe(self, data: Data):
         Raises:
             requests.HTTPError: If the API request fails.
         """
-        self._nyx_delete(f"{NYX_PURCHASES_TRANSACTIONS_ENDPOINT}/{data.creator}/{data.name}")
+        # Double encode `/` for community mode
+        creator = quote_plus(quote_plus(data.creator))
```

Comment:
`requests` will encode already so shouldn't this be just a single encode here?

Reply:
https://github.com/Iotic-Labs/nyx-sdk/pull/32/commits/3987c5747188b30af4058e61464125f611458528